validator/specs/lib/validateTilesetSpec.js

```diff
@@ -272,3 +330,26 @@ function createSampleTileset(tileBoundingVolume, contentBoundingVolume) {
     };
     return sampleTileset;
 }
+
+function createContentlessSampleTileset(parentBoundingVolume, childBoundingVolume, childTransform) {
```

Comment:
A better name might be `createParentChildTileset`. It should also take in `parentTransform`.